### PR TITLE
Set up controller method to add spa sessions 

### LINKED ISSUE #5: [1pt] Set up controller method to add spa session
- [x]  Create add method to the Session controller

### PR DESCRIPTION
**Pull request by Nasirkhan294**
### In this milestone I:

- [ ] Prepared session_params private method for all attributes
- [ ] Prepared create method to call the session_params method to create a spa session

### CODE REVIEW

File: app/controllers/api/items_controller.rb [hunk 1]
```diff
@@ -29,4 +29,29 @@ def show
...
+  end
+
+  private
+
+  def session_params
+    params.require(:session).permit(
+      :name,
+      :description,
+      :deposit,
+      :spa_session_fee,
```
Comment (geekelo) [inline app/controllers/api/items_controller.rb:50]: ### Changes Required: 
Kindly change sessions/session to items/item

At the moment session is related to user authentication. And spa_sessions are referred to as items.

Feel free to leave a comment if you need further clarification. Don't forget to tag me in the comment

**Review by geekelo — CHANGES_REQUESTED** [review]
### Weldone @Nasirkhan294 
- [x] You did a great job working on your tasks.
- [x] You have successfully created methods to add items.

File: app/controllers/api/items_controller.rb (hunk 1 above)
Comment (Nasirkhan294) [reply app/controllers/api/items_controller.rb:50]: Hi @geekelo,

Thanks for taking to review my code. I have changed and pushed it already please check it once more and let me know if still something requires a change.

**Review by geekelo — APPROVED** [review]
Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉 @Nasirkhan294

**Pull request merged**